src/tools/r2rdump/R2RDump.cs

```diff
@@ -190,6 +191,7 @@ private ArgumentSyntax ParseCommandLine(string[] args)
                 syntax.DefineOptionList("rp|referencepath", ref _options.ReferencePaths, "Search paths for reference assemblies");
                 syntax.DefineOption("isb|inlineSignatureBinary", ref _options.InlineSignatureBinary, "Embed binary signature into its textual description");
                 syntax.DefineOption("sb|signatureBinary", ref _options.SignatureBinary, "Append signature binary to its textual description");
+                syntax.DefineOption("sd|sortDebugInfo", ref _options.SortDebugInfo, "Sort Debug Info before displaying to allow diffing");
```

Comment:
Nit - sorting (normalizing) the output for easier comparison is an idiom that occurs in other places in R2RDump too and is generally driven by the "-normalize" flag - is there any compelling reason why we need "sortDebugInfo" as a separate flag?

Reply:
I just didn't know `--normalize` existed :) I will switch to use `--normalize` instead, thanks!